e2e/rbd.go

```diff
@@ -141,6 +142,19 @@ var _ = Describe("RBD", func() {
 				createRBDStorageClass(f.ClientSet, f, make(map[string]string))
 			})
 
+			By("create a PVC and Bind it to an app with encrypted RBD volume with Vault KMS", func() {
```

Comment:
tests will fail as there are no deployment file changes for 1.13

Reply:
done